ADAL/src/cache/ADTokenCache.m

```diff
@@ -356,11 +356,29 @@ -(NSMutableArray*)filterOutTombstones:(NSArray*) items
         {
             [itemsKept addObject:item];
         }
+        else
```

Comment:
I realised that getItemsWithKey:userId: is a better place to add the tombstone deletion code! :)